app/javascript/packages/document-capture/context/upload.tsx

```diff
@@ -0,0 +1,148 @@
+import { createContext } from 'react';
+import { useObjectMemo } from '@18f/identity-react-hooks';
+import defaultUpload from '../services/upload';
+
+const UploadContext = createContext({
+  upload: defaultUpload,
+  getStatus: () => Promise.reject() as () => Promise<UploadSuccessResponse>,
+  statusPollInterval: undefined as number,
+  isMockClient: false,
+  backgroundUploadURLs: {} as Record<string, string>,
+  backgroundUploadEncryptKey: undefined as CryptoKey,
+  flowPath: 'standard' as FlowPath,
+  csrf: null as string,
+});
+
+UploadContext.displayName = 'UploadContext';
+
+type ReactNode = import('react').ReactNode;
+
+type FlowPath = 'standard' | 'hybrid';
+
+// Upload field error, after normalized to error instance.
```

Comment:
Can we use JSDoc for all of these comments? I think it's required to be picked up as associated with the thing it's documenting (e.g. in VSCode hover tooltips).

```suggestion
/**
 * Upload field error, after normalized to error instance.
 */
```

Same applies to all interfaces and property descriptions.

Reply:
Yes, will update them 👍